psutil/_pslinux.py

```diff
@@ -785,6 +785,20 @@ def net_if_stats():
 
 # --- disks
 
+def disk_block_stats(device):
+    """Returns a tuple of (ios_in_flight, time_in_queue_ms) for a disk."""
+    try:
+        with open_text("/sys/block/%s/stat" % device) as f:
+            line = f.readlines()[0]
+        fields = line.split()
+        (reads, read_merges, read_sectors, read_ticks,
+         writes, write_merges, write_sectors, write_ticks,
+         in_flight, io_ticks, time_in_queue) = map(int, fields[:11])
```

Comment:
I'm wondering whether it makes sense to return also `io_ticks`. I would be OK with that if `iostat -x` returns it but I'm not sure what's the column of reference (`svctm` perhaps?).